### DISCUSSION TITLE
Sample Print SCP

### DISCUSSION THREAD

**Discussion by jeffersonarar** [fo-dicom/fo-dicom#1383] [Q&A]
Good afternoon my dears!

We are testing the Sample Print SCP, we selected the printer and passed the name correctly, but it gives the error that appears in the image below.
We tested in version 4.0.0 ok
Only in the latest version that didn't work.
Message = "Cannot cast to 'Image'; type must be assignable from 'Byte[]'".

**Comment by gofal** [comment]
you have to reference the fo-dicom.imaging.desktop nuget package and register the WinformsImageManager. See https://fo-dicom.github.io/dev/v5/usage/image_rendering.html
